_includes/cloud-query/cloud-query-execute.md

```diff
@@ -2,5 +2,5 @@ There are two methods for executing a query in the UI:
 
 |Method | Instructions  |
 | --- | ----------- |
-| Click to run | Click the button in the top left labeled **RUN** and select your database.  |
+| Click to run | Click the button in the top left labeled **RUN**  |
```

Comment:
The structure of instructions works like this:

<do something> <to this> <which achieves something else>

So in this case all you say is "Click Run to execute the query."

Additionally, we don't use words to the effect "Click the button labelled..." because it's too many words and we'd need to do that for every single button or field in the product.

If there's a button called "X" the user "clicks the X button"
If there's a dropdown called "Y" the user "Clicks the Y drop-down"
etc...